.github/workflows/checks.yml

```diff
@@ -15,6 +15,12 @@ jobs:
         ]
 
     runs-on: ${{ matrix.os }}
+    env:
+      STARKNET_JVM_INTEGRATION_TEST_SOURCE: "env"
+      STARKNET_JVM_SKIP_INTEGRATION_GETTERS_TEST: "false"
+      STARKNET_JVM_RPC_URL: ${{ secrets.STARKNET_JVM_RPC_URL }}
+      STARKNET_JVM_INTEGRATION_ACCOUNT_ADDRESS: ${{ secrets.STARKNET_JVM_INTEGRATION_ACCOUNT_ADDRESS }}
+      STARKNET_JVM_INTEGRATION_PRIVATE_KEY: ${{ secrets.STARKNET_JVM_INTEGRATION_PRIVATE_KEY }}
```

Comment:
> I think we can drop the STARKNET_JVM prefix.

We could, but if those are to be used locally, it would be confusing. What if at some point similar approach is used in starknet.py or starknet.swift?

> btw I was under the impression, that we'll be only running getter tests on CI. If so, why we need an account and private key?

Those tests can be used both locally and on CI. We will need an account if we want to make any tests that require gas, this is the account used in starknet.py that already has gas.

Reply:
> We could, but if those are to be used locally, it would be confusing. What if at some point similar approach is used in starknet.py or starknet.swift?

Very slim chances for that. And even if, I don't think running those with the same account is a bad thing.

> Those tests can be used both locally and on CI. We will need an account if we want to make any tests that require gas, this is the account used in starknet.py that already has gas.

Please, do not run anything on CI that needs funds. This is bad. What if the funds dry out? We have failing tests and a headache. CI should only do things, that does not require an account. Locally, it's fine.

Reply:
Discussed offline, clarification is:

on CI there will be no tests that require any funds. an account will only be used to sign fee estimation transaction, which does nto require any fee.